package.json

```diff
@@ -19,17 +21,17 @@
     "killApp": "(lsof -t -i:7080 | xargs kill)",
     "killStorybook": "(lsof -t -i:9001 | xargs kill)",
     "lighthouse": "lighthouse http://localhost:7080/news/articles/c85pqyj5m2ko --output-path=./reports/report.html --view",
-    "postshrinkwrap": "if test -z $CI; then \n npm run force-package-lock-https \n fi",
+    "postshrinkwrap": "if test -z $CI; then \n npm run force-package-lock-https \n fi; git update-index --assume-unchanged .env",
     "speculate": "speculate",
     "start": "NODE_ENV=production node build/server.js",
     "storybook": "start-storybook -p 9001 -c .storybook",
     "test": "npm run test:lint && npm run dataValidate && npm run test:unit && npm run test:ampValidate:ci",
-    "test:accessibility": "npm run killApp && npm run build:local && run-p --race start bbcA11y",
+    "test:accessibility": "npm run killApp && npm run build && run-p --race start bbcA11y",
     "test:ampValidate": "npm run killApp && npm run build -- --hide-modules --colors && run-p --race start amp:validate",
```

Comment:
For this PR we need to be verrry careful to find every `npm run build` command and ensure it changes to npm run build:ci/test/live

Reply:
I've checked for the previous `build:local` changes in that PR. I changed all the npm scripts and the travis config. I've also updated https://github.com/bbc/simorgh-infrastructure/pull/261 which is another `build` command. 

Can you think of any others?

Reply:
Found a few more with a search 🙈